src/fides/api/ops/models/privacy_request.py

```diff
@@ -190,6 +191,7 @@ class PrivacyRequest(IdentityVerificationMixin, Base):  # pylint: disable=R0904
 
     cancel_reason = Column(String(200))
     canceled_at = Column(DateTime(timezone=True), nullable=True)
+    consent_preferences = Column(MutableList.as_mutable(JSONB), nullable=True)
```

Comment:
We need somewhere to store this, so I'm storing as a List of Consent objects.  These pertain to the preferences submitted for a given request which is why we need to persist them separately from the current saved preferences.